act-rules-format.bs

```diff
@@ -83,58 +83,43 @@ Even within a rules group, certain rules may not be used. This leaves one fewer
+ACT Rules can operate on different aspects of the subject being tested. An aspect is a distinct part of the test subject itself or its underlying implementation. For example, rendering a web page to an end user involves multiple different technologies, some or all of which may be of interest to an ACT Rule. Some rules may need to operate directly on the HTTP messages exchanged between a server and a client, while others need to operate on the DOM tree exposed by a web browser. Some rules may even need to operate on several aspects simultaneously such as both the HTTP messages and the DOM tree.
 
-The following test subject types are common in accessibility testing:
-
-- HTTP Response
-- DOM Tree
-- Rendered Page
-- Template
-- Script
-
-Other types of test subjects may be possible. In those cases the ACT Rule MUST include a detailed description of the test subject type, or a reference to the that description.
+Each ACT Rule MUST include a description of all the aspects under test by the rule. Each aspect MUST be discrete and no overlap exist between aspects. Some aspects are already well defined within the context of web content, such as HTTP messages, DOM tree, and CSS styling, and do not warrant a detailed description. Other aspects may not be well defined or even specific to web content. In these cases, an ACT Rule SHOULD include a detailed description, or a reference to one, of the aspect in question.
 
+Common Aspects {#input-aspects-common}
+--------------------------------------
 
-HTTP Response Testing {#input-types-http-response}
---------------------------------------------
+### HTTP Messages ### {#input-aspects-http}
 
-Testing the files as they are served to the web browser (or other user agent) has its limitations. The files may be manipulated in different ways through presentation and scripting. Although this is an excellent place for parser testing.
+The HTTP messages exchanged between a client and a server as part of requesting a web page may be of interest to ACT Rules that wish to perform validation of e.g. HTTP headers or unparsed HTML and CSS.
 
+### DOM Tree ### {#input-aspects-dom}
 
-DOM Tree Testing {#input-types-dom-tree}
------------------------------------
+The DOM tree constructed from parsing HTML, and optionally executing DOM manipulating JavaScript, may be of interest to ACT Rules that wish to test the structure of web pages. In the DOM tree, information about individual elements of a web page, and their relations, becomes available.
```

Comment:
I would remove the phrase "wish to" from the first sentence.  The rules don't "wish". ;)

Also to be more clear the last sentence suggest: "...information about individual elements of a web page, and the relationships between them, becomes available."  Or am I missing the correct meaning?